### PR TITLE
Partial config save

### PR DESCRIPTION
**Pull request by Miodec**
Added support for partial config save to avoid constantly sending full config all the time

### CODE REVIEW

File: backend/dao/config.ts [hunk 1]
```diff
@@ -3,9 +3,16 @@ import db from "../init/db";
 
 class ConfigDAO {
   static async saveConfig(uid, config): Promise<UpdateResult> {
+    //https://stackoverflow.com/questions/10290621/how-do-i-partially-update-an-object-in-mongodb-so-the-new-object-will-overlay
+    const temp = {};
```
Comment (Bruception) [inline backend/dao/config.ts:7]: Refactor:

```js
_.mapKeys(config, (value, key) => `config.${key}`);
```

**Comment by Ferotiq** [comment]
I’ll probably only be able to review this when I get home, so about another 4 hours.

File: frontend/src/scripts/controllers/preset-controller.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export function apply(_id: string): void {
   const snapshot = DB.getSnapshot();
   snapshot.presets?.forEach((preset) => {
     if (preset._id == _id) {
-      Config.apply(JSON.parse(JSON.stringify(preset.config)));
+      UpdateConfig.apply(JSON.parse(JSON.stringify(preset.config)));
```
Comment (Ferotiq) [inline frontend/src/scripts/controllers/preset-controller.ts:12]: what’s with the parse and stringify?

Comment (Miodec) [reply frontend/src/scripts/controllers/preset-controller.ts:12]: Iiiii... dont remember been like this for a while i think

Comment (Ferotiq) [reply frontend/src/scripts/controllers/preset-controller.ts:12]: Odd, try removing it and see if it breaks something.

Comment (Miodec) [reply frontend/src/scripts/controllers/preset-controller.ts]: Works fine

**Review by Bruception — APPROVED** [review]
LGTM IIW

**Pull request merged**